### DISCUSSION TITLE
Trouble with removing a module from page

### DISCUSSION THREAD

**Discussion by MikeKilmer** [DNNCommunity/Dnn.CommunityForums#132] [Q&A]
I added a existing module to a page not understanding it was a copy of a module on another page. Then you changed the code and found it changed on the other page also. I change it back but now don't know how to remove the module from the page I added it too. There is no edit or remove icon on the module to do anything.. How can I remove it from the page I added it to??? Thank
![no menu in drop down](https://user-images.githubusercontent.com/94575582/167268876-93b8112a-f319-4ba5-a1c1-2e471a01d410.jpg)
s

**Comment by WillStrohl** [comment]
Hello...  I'm not sure what's going on here.  Can you please verify the module you're using and moving, including the version?  Also, in order for us to know how to potentially provide an answer, a clear "steps to reproduce" would be very helpful.  Right now, I'm not sure what the issue here is and how to provide any other feedback.  Sorry.  :(

**Reply by MikeKilmer** [reply]
The version is 08.00.04 (226)
Module is htmlmodule.ascx

I was adding some video lightbox type video icons to the bottom of a page so I used an existion module from another page. I thought it just copied the module but when you do it that way it appearently it copys the module as a global and when you change one it changes the other. The pencil or setting icon are there but do not open on that page. But they do open on the page I copied it from. So because of that I cannot click on a delete dropdown in the settings.

**Reply by WillStrohl** [reply]
Ah...  That sounds like the HTML module, maybe.  Does that sound right to you?  You've posted in the Community Forums discussion board.  This isn't the forums for the community, but rather a module that's named "community forums" that any website owner can use to create and manage discussions on their respective DNN websites.  

If you want to have a discussion about how to do something in DNN, you should instead [use the forums on the official DNN community website](https://dnncommunity.org/forums/aff/8).  

This area is meant to help people that are using or helping to build the Community Forums module.  :)
